apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -154,9 +153,6 @@ def render(element_html: str, data: pl.QuestionData) -> str:
             else:
                 raise unreachable
 
-        case panel:
```

Comment:
this isn't needed as strict mode already errors on this https://github.com/microsoft/pyright/discussions/5186#discussioncomment-6017087.